### ISSUE TITLE
No log displays on Activities after changing access policies for one file share

### ISSUE THREAD

**Issue by v-limu** [microsoft/AzureStorageExplorer#1837]
**Storage Explorer Version:** 1.10.0
**Build:** 20190910.8
**Branch:** master
**Platform/OS:** Windows 10/Linux Ubuntu 18.04/macOS High Sierra
**Architecture:** ia32/x64
**Regression From:** Not a regression

**Steps to reproduce:**

1. Expand one storage account -> File Shares -> Create a new file share.
2. Right click the file share -> Click 'Manage Access Policies...' on the content menu.
3. Add an access policy and click 'Save'.
4. Check whether there shows log message on Activities or not.

**Expect Experience:**
The log displays in Activities. 
![image](https://user-images.githubusercontent.com/54055206/64599512-120b8c00-d3ec-11e9-9974-6a2c7e78f175.png)

**Actual Experience:**
No log displays on Activities.

**More Info:**

1. This issue also reproduces for one table/queue.
2. This issue doesn't reproduces for one blob container.

**Comment by MRayermannMSFT** [comment]
Chose to solve this as part of moving this to V12 SDK. Currently blocked by SDK bug:

https://github.com/Azure/azure-sdk-for-js/issues/13540

**Comment by MRayermannMSFT** [comment]
Fixed merged into main. Activities should be roughly the same as for blob.
